elixir/apps/web/lib/web/live/sign_in.ex

```diff
@@ -200,31 +200,46 @@ defmodule Web.SignIn do
         placeholder="Enter your email"
         required
       />
-
-      <:actions>
-        <.button phx-disable-with="Sending..." class="w-full">
-          Request sign in token
-        </.button>
-      </:actions>
+      <.button phx-disable-with="Sending..." class="w-full" style="info">
+        Request sign in token
+      </.button>
     </.simple_form>
     """
   end
 
   def openid_connect_button(assigns) do
     ~H"""
-    <a href={~p"/#{@account}/sign_in/providers/#{@provider}/redirect?#{@params}"} class={~w[
-          w-full inline-flex items-center justify-center py-2.5 px-5
-          bg-white rounded
-          text-sm text-neutral-900
-          border border-neutral-200
-          hover:bg-neutral-100 hover:text-neutral-900
-    ]}>
-      Sign in with <%= @provider.name %>
-    </a>
+    <.button
+      navigate={~p"/#{@account}/sign_in/providers/#{@provider}/redirect?#{@params}"}
+      class="w-full"
+      style="info"
+    >
+      <%= provider_icon(@provider) %> Sign in with&nbsp;<strong><%= @provider.name %></strong>
+    </.button>
     """
   end
 
   def adapter_enabled?(providers_by_adapter, adapter) do
     Map.get(providers_by_adapter, adapter, []) != []
   end
+
+  defp provider_icon(%{adapter: :google_workspace} = assigns) do
+    ~H"""
+    <img src={~p"/images/google-logo.svg"} alt="Google Workspace Logo" class="w-5 h-5 mr-2" />
+    """
+  end
+
+  defp provider_icon(%{adapter: :openid_connect} = assigns) do
+    ~H"""
+    <img src={~p"/images/openid-logo.svg"} alt="OpenID Connect Logo" class="w-5 h-5 mr-2" />
+    """
+  end
+
+  defp provider_icon(%{adapter: :microsoft_entra} = assigns) do
+    ~H"""
+    <img src={~p"/images/entra-logo.svg"} alt="Microsoft Entra Logo" class="w-5 h-5 mr-2" />
```

Comment:
If we do this we also might want to show this logo elsewhere in the UI, eg. on the IdP list or creation.

Reply:
I was also thinking that. Added it as a core component and added it to the Identity Providers -> New form:

<img width="764" alt="Screenshot 2024-02-05 at 9 45 38 AM" src="https://github.com/firezone/firezone/assets/167144/b8f8d5ae-37c1-4a7e-88ba-a0e3b9551744">